app/components/nav-bar.jsx

```
@@ -0,0 +1,39 @@
+import React from 'react';
+import TextButton from './text-button';
+import FilledButton from './button';
+import SearchBox from './search-box';
+// import ToggleIconButton from './toggle-icon-button';
+
+export default function NavBar() {
+    return (
+        <div className="nav-bar flex flex-grow-0 pl-4 md:pl-0 pt-3 items-center justify-between w-full top-0 max-h-24 sticky bg-inherit z-[1000]" autoFocus={true}>
```

Biome

[error] Avoid the autoFocus attribute.

Unsafe fix: Remove the autoFocus attribute.


(lint/a11y/noAutofocus)